topicExtractor.py

```diff
@@ -10,11 +10,13 @@
 
```

Comment:
My Thought shared here https://github.com/tl-its-umich-edu/annoto-gai/pull/16/files#r1611987954 can be applied to this class well.